src/libxtp/tools/molpol.cc

```diff
@@ -44,20 +44,10 @@ void MolPol::Initialize(const tools::Property& user_options) {
       ".mpsoutput", _job_name + "_polar.mps");
   _polar_options = options.get(".options_polar");
 
-  bool target_exists = options.exists(".target");
-  bool qmpackage_exists = options.exists(".qmpackage");
-  if (target_exists && qmpackage_exists) {
-    throw std::runtime_error(
-        "Can only read either from target or qmpackage logfile");
-  }
-
-  if (!target_exists && !qmpackage_exists) {
-    throw std::runtime_error(
-        "You have to define a polar targer <target> or a or qmpackage logfile");
-  }
-
-  if (target_exists) {
+  // polar targer or qmpackage logfile
+  const std::string& mode = options.get("mode").as<std::string>();
 
+  if (mode == "target") {
```

Comment:
"target" is not a good name for the mode. Maybe "explicit polarisation"

Reply:
I think that `file` is more meaningful because the help message says *Read the polar target from a file or a qmpackage logfile*

Reply:
yeah the help message is wrong, "Read the target from a qmpackage logfile or enter it exlicitly"

Reply:
Which was most likely my doing